lib/inspec/formatters/base.rb

```diff
@@ -23,6 +23,7 @@ def dump_summary(summary)
       run_data[:version] = Inspec::VERSION
       run_data[:statistics] = {
         duration: summary.duration,
+        controls: statistics,
```

Comment:
Lets not add stats as default to our json reporter.

Reply:
Ill remove it from the json report so it never shows up but leave this here for now for the exit code info.

Reply:
@jquick @chris-rock what data is in statistics now that we have removed the grouping? If we are going to have the exit codes which use the statistics data - I think it is reasonable to have it. One stat we use is the 'timestamp' of the start of the run